Configuration/Generator/python/TestEnergy6500GeV_cfi.py

```diff
@@ -0,0 +1,22 @@
+import FWCore.ParameterSet.Config as cms
+
+generator = cms.EDProducer("FlatProtonLogKsiLogTGun",
+    LogKsiDistShape = cms.untracked.bool(True),
+    LogTDistShape = cms.untracked.bool(True),
+    GenerateMirroredProtons = cms.untracked.bool(False),
+
+    RightArm = cms.untracked.bool(True),
+    LeftArm = cms.untracked.bool(True),
+    NominalEnergy = cms.untracked.double(6500.0), # [GeV]
```

Comment:
Add comment that:
 - center of mass energy is 13 TeV and nominal energy is 6.5 TeV